config/tgl-cavs.toml

```diff
@@ -388,3 +388,18 @@ count = 15
 	mod_cfg = [0, 0, 0, 0, 960, 488500, 16, 16, 0, 0, 0,
 			1, 0, 0, 0, 960, 964500, 16, 16, 0, 0, 0,
 			2, 0, 0, 0, 960, 2003000, 16, 16, 0, 0, 0]
+        [[module.entry]]
+        name = "RTC_AEC"
+        uuid = "B780A0A6-269F-466F-B477-23DFA05AF758"
+        affinity_mask = "0xF"
+        instance_count = "2"
```

Comment:
I still fail to understand why this is a mandatory field. In a truly versatile firmware this should only be enforced when needed? And this also implies the topology cannot be trusted.

Reply:
@cujomalainey please check my comment above. Generally for 3rd party components we can make it optional with assumption that if no instance_count is defined then it is unlimited.